react-effects/my-app/src/List.jsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable @typescript-eslint/no-unused-vars -- Remove me */
+import { useEffect, useState } from 'react';
+import readItems from './read';
+
+export default function List() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [items, setItems] = useState([]);
+  const [error, setError] = useState();
+
+  // Your code here:
+  useEffect(() => {
+    readItems()
```

Comment:
This works, but you should do it with async/await since that is the more modern approach to async JavaScript.